examples/webpack.config.local.js

```diff
@@ -77,15 +79,11 @@ function makeLocalDevConfig(EXAMPLE_DIR = LIB_DIR, linkToLuma) {
         {
           // Compile source using buble
           test: /\.js$/,
-          loader: 'buble-loader',
-          include: [`${ROOT_DIR}/modules`],
+          loader: 'babel-loader',
           options: {
-            objectAssign: 'Object.assign',
-            transforms: {
-              dangerousForOf: true,
-              modules: false
-            }
-          }
+            presets: ['@babel/env']
```

Comment:
Fine to land.

That said it could be good to add some comment here as to why we are transpiling. As far as I understand, it is only related to some issue with transpiled/non-transpiled class inheritance that only happens in custom layer examples, it doesn't affect e.g. layer-brower. 

Presumably there is one specific transform we need to get around that. On the other hand, preset-env does not guarantee any specific transform (I am actually surprised that "class inheritance" is included by present-env even with our current browser version settings, and I was thinking we would bump those soon).